packages/@expo/cli/src/start/server/metro/MetroBundlerDevServer.ts

```diff
@@ -70,7 +70,9 @@ export class MetroBundlerDevServer extends BundlerDevServer {
     // https://github.com/expo/expo/issues/13114
     prependMiddleware(middleware, manifestMiddleware);
 
-    middleware.use(new InterstitialPageMiddleware(this.projectRoot).getHandler());
+    middleware.use(
+      new InterstitialPageMiddleware(this.projectRoot, options.location.scheme).getHandler()
```

Comment:
The other middleware uses the format of `projectRoot, { options as object }`

Reply:
The `scheme` here is capable of being dynamic, seems like we shouldn't pass in the default value.

Reply:
Could you point me to where the scheme should come from then? Do you mean retrieve it using `getOptionalDevClientSchemeAsync` inside of the InterstitialPageMiddleware (and fall back to this value if that returns null)?

Reply:
I recommended a TODO, for now. I can implement this later since it's an edge case they may only happen theoretically.